printer_mods/techprolet/2020_aplic_cam_mount/README.md

```diff
@@ -0,0 +1,41 @@
+## 2020 Aplic Cam Mount
+
+![2020 Aplic Cam Mount installation example](img/2020_aplic_cam_mount.jpg)
+**2020 Aplic Cam Mount** is a 3D printed part that is used to mount a cheapo *aplic hd camera* (or similar) to the 2020 frame at bed level. The mount is designed to be held in place with a M3 screw and a M3 nut. Additionaly, you need 4x M3 heat inserts, 4x M3 spacers and 4x M3x6 screws for mounting the camera on the mount (pun intended).
+
+
+You can also invert mount the camera under the gantry, which would work better for monitoring higher layers. *Photo courtesy of _tsimo_ V2.6592*
+
+![2020 Aplic Cam Mount installation example](img/2020_aplic_cam_inverse_mount.jpg "Inverse (gantry) mount. Photo courtesy of `_tsimo_` V2.6592")
+### Files
+* `aplic_cam_mount_voron_2_4_300x300.stl` - The mount STL, angled for the Voron 2.4 300x300mm geometry.
+* `aplic_cam_mount_voron_2_4_300x300_shortened.stl` - Shortened version STL,for inverse mounting on a Voron 2.4 300x300mm gantry.
+* `aplic_cam_mount.scad` - The OpenSCAD file for the mount (you can modify the parameters for your own printer's geometry).
+
+### Parts
+The mount is designed to be used with the following parts: 
+```
+- 1x 3d printed 2020 Aplic Cam Mount
+- 1x Aplic Webcam - 1920x1080P stripped from its plastic case
+- 1x M3x10 screw
+- 1x M3 2020T-nut
+- 4x M3 heat insertsd
+- 4x M3 spacer
```

Comment:
I assume it doesn't really matter what length these are, but it would be a good idea to mention a recommended length or min/max length for these.